src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -6790,6 +7007,250 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    delete:
+      x-alfresco-since: "7.4"
+      tags:
+        - tags
+      summary: Delete a tag
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.4 and newer versions.
+
+        Deletes the tag with **tagId**. This will cause the tag to be removed from all nodes.
+      operationId: deleteTag
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/tagIdParam'
+      responses:
+        '204':
+          description: Successful response
```

Comment:
Can we have `400` error code? For example, see `  DELETE methods of /nodes/{nodeId}/targets/{targetId}` or  `  /nodes/{nodeId}/versions/{versionId}:`

I know some of our existing APIs don't have this, but we can sort them out later.

Reply:
I've updated all the methods to have 400 status codes for tags and categories. I have made them all have a generic statement since the existing approach has not been easy to maintain.
```
        '400':
          description: An invalid parameter has been supplied.
```